icu4c/source/common/unicode/uconfig.h

```diff
@@ -323,6 +323,16 @@
 #   define UCONFIG_NO_NORMALIZATION 0
 #endif
 
+/**
+ * \def UCONFIG_NO_CJ_ML_PHRASE_BREAKING
+ * This switch turns off BudouX's phrase breaking, which would make ICU use the dictionary
+ * based solution.
+ *
+ */
+#ifndef UCONFIG_NO_CJ_ML_PHRASE_BREAKING
+#   define UCONFIG_NO_CJ_ML_PHRASE_BREAKING 0
```

Comment:
This needs to be changed to 1 so that the default behavior does not change.

And in order to test the new code regularly, we need an continuous-integration test that turns this flag off so that the new code is active.

@FrankYFTang did this for another BreakIterator ML feature before. Maybe we can turn on both flags in the same test workflow? Otherwise add a new one.

Reply:
There is a prebuild error.
_Run python3 tools/scripts/uconfig_vars_test.py -u
No definition for flag UCONFIG_NO_CJ_ML_PHRASE_BREAKING found!_

It looks like it could only be '0'.
Does ICU have naming convention that should start with "UCONFIG_NO"?
https://github.com/unicode-org/icu/blob/main/tools/scripts/uconfig_vars_test.py#L76

I change the flag name to make it build pass.